ledger/src/bigtable_upload.rs

```diff
@@ -61,7 +64,8 @@ pub async fn upload_confirmed_blocks(
         .collect();
 
     if blockstore_slots.is_empty() {
-        return Err(format!("Ledger has no slots from {starting_slot} to {ending_slot:?}").into());
+        info!("Ledger has no slots from {starting_slot} to {ending_slot:?}");
```

Comment:
I considered a higher log level here. Wdyt?

Reply:
Definitely not error; I do see some merit to promoting to warn tho. Ie, if someone chooses a bad value, the warn's might be more attention grabbing that it is operating on a range where nothing will actually get uploaded. Additionally, if someone has a (large enough) gap for whatever reason (maybe someone's warehouse node goes down and they don't have `--no-spapshot-fetch` for the restart), the warns might be an earlier signal of that.